dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/imports/TrackerImportController.java

```diff
@@ -156,6 +162,6 @@ public TrackerImportReport getJobReport( @PathVariable String uid,
             .getJobSummaryByJobId( JobType.TRACKER_IMPORT_JOB, uid ) )
             .map( report -> trackerImportService.buildImportReport( (TrackerImportReport) report,
                 trackerBundleReportMode ) )
-            .orElseThrow( () -> new NotFoundException( uid ) );
```

Comment:
The `NotFoundException(String)` constructor was not expecting a message but a `uid` and compose a message for it. This makes it impossible to use the exception with other messages. Therefore I created the `notFoundUId` factory method which does that so that the `NotFoundException(String)` can accept a _plain_ message. This was the only place where the `NotFoundException(String)` was used before so this was the only place that now needed to use `notFoundUid`.